x-pack/plugin/logsdb/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/LogsUsageRollingUpgradeIT.java

```diff
@@ -31,8 +30,8 @@ public LogsUsageRollingUpgradeIT(@Name("upgradedNodes") int upgradedNodes) {
         super(upgradedNodes);
     }
 
-    @Before
-    public void checkFeatures() {
+    @Override
+    protected void beforeUpgrade() {
         if (Build.current().isSnapshot()) {
```

Comment:
I think we should use `getOldClusterVersion()` and check whether suffix is `-SNAPSHOT`?

I don't think this works, because this code runs in the jvm that runs the test suite and not Elasticsearch.